contracts/DebtLocker.sol

```diff
@@ -57,7 +66,7 @@ contract DebtLocker is IDebtLocker, DebtLockerStorage, MapleProxied {
         _principalRemainingAtLastClaim = loan_.principal();
     }
 
-    function setFundsToCapture(uint256 amount_) override external {
+    function setFundsToCapture(uint256 amount_) external override live {
```

Comment:
Setters should be allowed during paused state, since:
- there is to harm setting values during a paused state (and so no need to also waste gas during live states)
- the fix might require a setter, during a paused state